osu.Game.Rulesets.Mania/Edit/ManiaHitObjectComposer.cs

```diff
@@ -27,6 +29,8 @@ public ManiaHitObjectComposer(Ruleset ruleset)
         protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnlyDependencyContainer parent)
             => dependencies = new DependencyContainer(base.CreateChildDependencies(parent));
 
+        public Column ColumnAt(Vector2 screenSpacePosition) => ((ManiaPlayfield)RulesetContainer.Playfield).GetColumnByPosition(screenSpacePosition);
```

Comment:
another case of deep pathing we may want to avoid.

Reply:
How would you avoid this one? It's in `ManiaHitObjectComposer`

Reply:
As in potentially we want the method `GetColumnByPosition` at a `ManiaRulesetContainer` level.